bin/build-nodes-json

```diff
@@ -63,11 +63,12 @@ def main():
         auth_url=auth_url
     )
 
+    bmc_id = [ s.id for s in nova.servers.list() if bmc_base in s.name ]
```

Comment:
I'd prefer to stick with .startswith() like the original version used.  Just using "in" can still result in incorrect matches.  Consider two environments, one with a bmc_base of "bmc" and another with "bmc-2".  build-nodes-json against "bmc" would match both.